roles/pulp-devel/tasks/install_podman.yml

```diff
@@ -28,4 +28,19 @@
   args:
     creates: /etc/containers
   when: ansible_distribution == 'Debian'
+
+# Needed to allow non-root users to run podman containers on Debian
+- name: Enable unprivileged_userns_clone (Debian specific)
+  sysctl:
+    name: kernel.unprivileged_userns_clone
+    value: "1"
+  when: ansible_distribution == 'Debian'
+  ignore_errors: true
```

Comment:
If anyone has an idea how to make the tests run without the `ignore_errors`, I am all ears.
As far as I can tell this task just won't work in a molecule container, since the relevant files are read only.
It works fine in Vagrant...

Reply:
The strict answer to your question is "Molecule will skip tasks which are tagged with either molecule-notest"
https://molecule.readthedocs.io/en/latest/configuration.html#molecule.provisioner.ansible.Ansible

However, I prefer a `stat` module check for /.dockerenv , and an additional when line. (a '-' in front of each when line instead of an "and"). I think we can assume that containers cannot modify sysctl, and this is the devel role anyway.
https://stackoverflow.com/a/25518345
(The old /bin/running-in-container no longer exists as of 16.04.)

Reply:
I went with `when: ansible_virtualization_type != 'docker'`. Has the advantage of using an Ansible fact that already exists, and appears to have the desired effect.

Reply:
@quba42 Cool, I just assumed ansible_virtualization_type would not work with containers since they can be on top of hypervisors. I would have run the setup module against a container, and looked at them. Thank you!